tempelis/config/parser.go

```diff
@@ -205,7 +205,7 @@ func mergeUsers(target map[string]string, source map[string]string, r Restrictio
 		if _, ok := target[k]; ok {
 			return fmt.Errorf("cannot overwrite users (duplicate user %s)", k)
 		}
-		if len(v) != 9 {
+		if len(v) != 11 {
```

Comment:
Would if work if this was 

```
if len(v) != 9 || len(v) != 11
```

Reply:
Ok I just saw the comment and documentation from above.

Reply:
It would but it would be safer to move toward an unlimited amount of characters and only focus on the `U` or the `W` which has been standard for some time

Reply:
but I think for your case we could just move to this to solve the immediate concern and then move to different case later. Thoughts?

Reply:
that would work. I have not seen any user ids with lengths of 10 yet.

Reply:
So if I add 
```
if len(v) != 9 || len(v) != 11
```
I can leave the test character length at 11?

Reply:
yes I would just add a test case for both cases